.github/ISSUE_TEMPLATE/config.yml

```diff
@@ -1,3 +1,4 @@
+# yamllint disable-line rule:document-start
```

Comment:
Do we need this, since it looks like `document-start` doesn't need to be present per the rules you defined?

Reply:
oh, Not needed anyone, it has already been disabled